packages/kit/src/views/ReferFriends/pages/YourReferred/index.tsx

```
@@ -166,7 +167,11 @@ function WalletList() {
                   )}
                 </SizableText>
               </ListItem>
-              <Divider mx="$5" borderBottomColor="$borderSubdued" />
+              <Stack
+                h={StyleSheet.hairlineWidth}
+                bg="$borderSubdued"
+                w="100%"
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)